### PR TITLE
DOC: Clean up small errors and warnings

### PR DESCRIPTION
**Pull request by bashtage**
Clean up small errors and warnings in doc build

### CODE REVIEW

File: statsmodels/tsa/vector_ar/vecm.py [hunk 1]
```diff
@@ -805,7 +805,9 @@ class VECM(tsbase.TimeSeriesModel):
     References
     ----------
     .. [1] Lutkepohl, H. 2005. *New Introduction to Multiple Time Series Analysis*. Springer.
-    .. [2] Johansen, S. 1995. *Likelihood-Based Inference in Cointegrated Vector Autoregressive Models*. Oxford University Press.
+
+    .. [2] Johansen, S. 1995. *Likelihood-Based Inference in Cointegrated *
+           *Vector Autoregressive Models*. Oxford University Press.
```
Comment (jbrockmendel) [inline statsmodels/tsa/vector_ar/vecm.py]: It looks like everywhere else you changed `[2]` to `[*]`.  Why is this different?

**Comment by bashtage** [comment]
Uncited reference create warnings.  [*] Is the correct way in Sphinx to add
a reference that isn't cited in the doc string.

File: statsmodels/tsa/statespace/dynamic_factor.py [hunk 2]
```diff
@@ -136,7 +136,7 @@ class DynamicFactor(MLEModel):
 
     References
     ----------
-    .. [1] Lutkepohl, Helmut. 2007.
+    .. [*] Lutkepohl, Helmut. 2007.
```
Comment (jbrockmendel) [inline statsmodels/tsa/statespace/dynamic_factor.py]: Should we standardized on the umlaut `Lütkepohl`?

**Comment by bashtage** [comment]
Do you mean ue or ü?  Does the latter potentially cause issues on Python
2.7?

**Comment by jbrockmendel** [comment]
In e.g. vecm we have the non-ascii version, which I assume is more-correct.  My understanding is that putting `# -*- coding: utf-8 -*-` takes care of the interpreter.

**Comment by josef-pkt** [comment]
about non-ASCII names

As long as the encoding is correctly specified `# -*- coding: utf-8 -*-` there should not be any problems, also not in python 2.
I thought we have some cases, but I don't find any in the modules that I checked as likely candidates.
`Lutkepohl` seems to be consistently spelled this way in vector_ar VAR and VECM. That's the closes mapping to ASCII. I'm not sure anybody will understand the German umlaut "ue" conversion, although For Hardle, I might be also using that instead of "Haerdle"
In references in docstrings we should switch to utf-8 and native spelling (but getting the utf-8 for non-ascii is not trivial when we don't have the corresponding keyboard).

e.g. random reference copied from my Zotero

Lütkepohl, Helmut, and Aleksei Netšunajev. 2017. “Structural Vector Autoregressions with Heteroskedasticity: A Review of Different Volatility Models.” Econometrics and Statistics 1 (January):2–18. https://doi.org/10.1016/j.ecosta.2016.05.001.

**Comment by josef-pkt** [comment]
jstor converts to simple ASCII

Hardle, W., J. Hart, J. S. Marron, and A. B. Tsybakov. 1992. “Bandwidth Choice for Average Derivative Estimation.” Journal of the American Statistical Association 87 (417):218–26. https://doi.org/10.2307/2290472.

statespace/mlemodel.py line 1905 uses the correct utf-8 "ü"

File: statsmodels/tsa/vector_ar/var_model.py [hunk 3]
```diff
@@ -1716,17 +1720,24 @@ def test_inst_causality(self, causing, signif=0.05):
...
+              The p-value corresponding to the test statistic.
+
+            "df" : float
+              The degrees of freedom of the Chi^2-distribution.
+
+            "conclusion" : str {"reject", "fail to reject"}
+              Whether H0 can be rejected or not.
+
+            "signif" : float
+              Signifigance level
```
Comment (josef-pkt) [inline statsmodels/tsa/vector_ar/var_model.py]: was this necessary
I thought the list with would work as rst

Comment (jbrockmendel) [reply statsmodels/tsa/vector_ar/var_model.py]: Signifigance --> Significance

**Comment by josef-pkt** [comment]
LGTM
I didn't know how to cleanup the references that are not cited.
properly spelled Lutkepohl looks good

(I will be back to merging in a few hours)

**Comment by jbrockmendel** [comment]
I don't know rst/sphinx well enough to give this a "LGTM" stamp (hoping jspkt recognizes that there is a non-trivial signal in places where I _do_ apply this stamp), but it does look internally consistent.

**Comment by bashtage** [comment]
The dictionary output was broken.  This is standard RST.

**Comment by bashtage** [comment]
Fixed.

**Comment by bashtage** [comment]
image

**Comment by bashtage** [comment]
LGTM

**Comment by josef-pkt** [comment]
@bashtage Thanks
@jbrockmendel  Thanks for reviewing

merged

**Pull request merged**